packages/api/tests/lambdas/test-cw-sf-execution-event-to-db.js

```diff
@@ -1,121 +0,0 @@
-'use strict';
```

Comment:
I'm not sure it can be undone now (at least my git-fu isn't that strong), but it's easier to track changes in a case like this if you do a `git mv test-cw-sf-execution-event-to-db.js test-cw-sf-event-to-db-records.js` instead of deleting one file and creating the other. That way you still get the git history instead of it looking like all of `event-to-db-records` was created in this PR.

Reply:
Exactly like was done for `packages/api/tests/lambdas/publish-reports/test-getGranuleRecordsFromCumulusMessage.js → packages/api/tests/models/granules/test-getGranuleRecordsFromCumulusMessage.js`. Eh, stuff happens.

Reply:
Yeah, I should have done that but not sure if it's easy to change now. I can check

Reply:
Since it's only a test file, don't spend too much time trying to change it.